yarn.lock

```diff
@@ -0,0 +1,6783 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We have a `package-lock.json` and a `yarn.lock`. Let's pick one and remove the other.